target/linux/realtek/dts/rtl931x.dtsi

```diff
@@ -296,6 +296,31 @@
 		};
 	};
 
+	pinmux@1b00103c {
+		compatible = "pinctrl-single";
+		reg = <0x1b00103c 0x4>;
+
+		pinctrl-single,bit-per-mux;
+		pinctrl-single,register-width = <32>;
+		pinctrl-single,function-mask = <0x1>;
+		#pinctrl-cells = <2>;
+
+		/* Enable GPIO 8, 9, 10 */
+		pinmux_disable_spi0: disable-spi0 {
+			pinctrl-single,bits = <0x0 0x0 0x400>;
+		};
+
+		/* Enable GPIO 12 */
+		pinmux_disable_spi0_cs1: disable-spi-cs1 {
+			pinctrl-single,bits = <0x0 0x0 0x200>;
+		};
+
+		/* Enable GPIO 11 */
+		pinmux_disable_spi0_cs0: disable-spi-cs0 {
+			pinctrl-single,bits = <0x0 0x0 0x100>;
+		};
+	};
+
```

Comment:
Make this a separate precondition PR and explain what it is about. Especially how it is (not) related to the NOR and NAND controllers in the SoC.

Reply:
Done, see #20263. Will rebase and de-draft this PR after #20263 was merged.